source/frontend/src/features/mapSideBar/disposition/tabs/offersAndSale/dispositionSale/hooks/useCalculateNetProceeds.tsx

```diff
@@ -2,12 +2,17 @@ import { getIn, useFormikContext } from 'formik';
 import { useEffect } from 'react';
 
 import { DispositionSaleFormModel } from '@/features/mapSideBar/disposition/models/DispositionSaleFormModel';
+import { SystemConstants, useSystemConstants } from '@/store/slices/systemConstants';
 
 export const useCalculateNetProceeds = (isGstEligible: boolean) => {
   const { values, touched, setFieldValue, isSubmitting } =
     useFormikContext<DispositionSaleFormModel>();
+  const { getSystemConstant } = useSystemConstants();
+  const gstConstant = getSystemConstant(SystemConstants.GST);
+  const gstDecimal = gstConstant !== undefined ? parseFloat(gstConstant.value) / 100 : undefined;
```

Comment:
Instead of undefined I thint it would be better if we use the exists() method given it also checks for null

Reply:
Updated